src/components/notifications/notifications.js

```diff
@@ -1,182 +1,181 @@
-        }
+    let body = item.Name;
 
-        var notification = {
-            title: 'New ' + item.Type,
-            body: body,
-            vibrate: true,
-            tag: 'newItem' + item.Id,
-            data: {}
-        };
+    if (item.SeriesName) {
+        body = item.SeriesName + ' - ' + body;
+    }
+
+    let notification = {
+        title: 'New ' + item.Type,
+        body: body,
+        vibrate: true,
+        tag: 'newItem' + item.Id,
+        data: {}
+    };
+
+    let imageTags = item.ImageTags || {};
+
+    if (imageTags.Primary) {
+        notification.icon = apiClient.getScaledImageUrl(item.Id, {
+            width: 80,
+            tag: imageTags.Primary,
+            type: 'Primary'
+        });
+    }
 
-        var imageTags = item.ImageTags || {};
+    showNotification(notification, 15000, apiClient);
+}
 
-        if (imageTags.Primary) {
-            notification.icon = apiClient.getScaledImageUrl(item.Id, {
-                width: 80,
-                tag: imageTags.Primary,
-                type: 'Primary'
-            });
-        }
+function onLibraryChanged(data, apiClient) {
+    let newItems = data.ItemsAdded;
 
-        showNotification(notification, 15000, apiClient);
+    if (!newItems.length) {
+        return;
     }
 
-    function onLibraryChanged(data, apiClient) {
-        var newItems = data.ItemsAdded;
-
-        if (!newItems.length) {
-            return;
-        }
+    // Don't put a massive number of Id's onto the query string
+    if (newItems.length > 12) {
+        newItems.length = 12;
+    }
 
-        // Don't put a massive number of Id's onto the query string
-        if (newItems.length > 12) {
-            newItems.length = 12;
-        }
+    apiClient.getItems(apiClient.getCurrentUserId(), {
 
-        apiClient.getItems(apiClient.getCurrentUserId(), {
+        Recursive: true,
+        Limit: 3,
+        Filters: 'IsNotFolder',
+        SortBy: 'DateCreated',
+        SortOrder: 'Descending',
+        Ids: newItems.join(','),
+        MediaTypes: 'Audio,Video',
+        EnableTotalRecordCount: false
 
-            Recursive: true,
-            Limit: 3,
-            Filters: 'IsNotFolder',
-            SortBy: 'DateCreated',
-            SortOrder: 'Descending',
-            Ids: newItems.join(','),
-            MediaTypes: 'Audio,Video',
-            EnableTotalRecordCount: false
+    }).then(function (result) {
+        let items = result.Items;
 
-        }).then(function (result) {
-            var items = result.Items;
+        for (const item of items) {
+            showNewItemNotification(item, apiClient);
+        }
+    });
+}
 
-            for (var i = 0, length = items.length ; i < length; i++) {
-                showNewItemNotification(items[i], apiClient);
-            }
-        });
-    }
+function getIconUrl(name) {
+    name = name || 'notificationicon.png';
+    return './components/notifications/' + name;
```

Comment:
Path should probably be relative to module itself, but I can't verify that - browser needs a secure connection (https?).

Reply:
There is no other icon than `notificationicon.png` in that folder, so this is probably dead code?

Reply:
There is also `badge.png` in the folder as well.

Reply:
`require.toUrl('.')` returns `./components/notifications?v=12`.
Let's leave it as it is now.